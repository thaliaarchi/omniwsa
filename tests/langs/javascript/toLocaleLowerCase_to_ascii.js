const locale = new Intl.Collator().resolvedOptions().locale;
console.log(`Characters which String.prototype.toLocaleLowerCase maps to ASCII with locale ${locale}:`);
for (let code = 0x80; code <= 0x10FFFF; code++) {
  const ch = String.fromCodePoint(code);
  const lower = [...ch.toLocaleLowerCase()];
  if (lower.every(ch => ch.codePointAt(0) <= 0x7f)) {
    console.log(`${ch} -> ${lower}`);
  }
}
