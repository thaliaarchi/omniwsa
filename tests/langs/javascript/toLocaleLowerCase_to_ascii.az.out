Node.js v23.3.0
Characters which String.prototype.toLocaleLowerCase maps to ASCII with locale az:
İ -> i
K -> k
